docs/tutorial.md

```diff
@@ -0,0 +1,198 @@
+
+```
+-IMG ?= controller:latest
++IMG ?= $(IMAGE_TAG_BASE):$(VERSION)
+```
+
+Once done, you do not have to set IMG or any other image variable in the CLI. The following command will build and push an operator image tagged as example.com/memcached-operator:v0.0.1 to Docker Hub:
+
+```
+make docker-build docker-push IMG=<registry>/<user>/<img-name>:<version>
+```
+
+### Generating CRD manifests 
+
+```sh
+$ mkdir manifest && \ 
+kustomize build config/default > manifest/manifest.yaml
+```
+
+### Update the `manifest.yaml` file
+
+Update image tag from `controller:latest` to `example.com/memcached-operator:v0.0.1`
+
+Then, remove `ClusterRole` with name `memcached-operator-manager-role` and `ClusterRoleBinding` with name `memcached-operator-manager-rolebinding`.
+
+Now, create the `ScopeTemplate` and add above removed `ClusterRole` in spec in `manifests/scopeTemplate.yaml`.
+
+```yaml
+apiVersion: operators.io.operator-framework/v1alpha1
+kind: ScopeTemplate
+metadata:
+  name: scopetemplate-sample-manager
+spec:
+  clusterRoles:
+  - generateName: manager-role
+    rules:
+    - apiGroups:
+      - "*"
+      resources:
+      - deployments
+      - memcacheds
+      verbs:
+      - create
+      - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
+    - apiGroups:
+      - cache.example.com
+      resources:
+      - memcacheds/finalizers
+      verbs:
+      - update
+    - apiGroups:
+      - cache.example.com
+      resources:
+      - memcacheds/status
+      verbs:
+      - get
+      - patch
+      - update
+    subjects:
+    - kind: ServiceAccount
+      name: memcached-operator-controller-manager
+      namespace: memcached-operator-system
+```
+
+Then, apply manifest and look for the pod logs.
+
+```
+$ kubectl apply -f manifest
+```
+
+The `memcached-operator` is running successfully with some errors. 
+
+Get the list of pods running under `memcached-operator-system` namespace.
+
+```
+$ kubectl get pods -n memcached-operator-system
+NAME                                                     READY   STATUS    RESTARTS   AGE
+memcached-operator-controller-manager-684554467c-4jflr   2/2     Running   0          9m45s
+```
+
+Check the pod logs and now you will be able to see that pod is complaining about `RBAC`.
+
+```
+$ kubectl logs memcached-operator-controller-manager-684554467c-4jflr manager -n memcached-operator-system
+```
+
+Create `ScopeInstance` that refers to `ScopeTemplate` and apply it.
+
+```
+cat << EOF | kubectl apply -f -
+apiVersion: operators.io.operator-framework/v1alpha1
+kind: ScopeInstance
+metadata:
+  name: scopeinstance-sample-manager
+spec:
+  scopeTemplateName: scopetemplate-sample-manager
+EOF
+```
+
+The `oria-operator` will create the `ClusterRole` and `ClusterRoleBinding`. Verify `ClusterRole` and `ClusterRoleBinding` with below commands.
+
+```
+$ kubectl get clusterroles
+
+$ kubectl get clusterrolebinding
+```
```

Comment:
I would change this to be more like:

---

```
kubectl get clusterroles
```

In the list you should see a `ClusterRole` with the name ...

```
kubectl get clusterrolebindings
```

In the list you should see a `ClusterRoleBinding` with the name ...

---